policy/LsstSimMapper.yaml

```diff
@@ -243,6 +243,35 @@ datasets:
   dcrCoadd_forced_metadata:
     template: dcrCoadd_forced_metadata/%(filter)s/%(tract)d/%(patch)s.boost
 
+  # Focalplane summary plots
+  focalplane_summary_fits:
+    level: Visit
+    persistable: MaskedImageF
+    python: lsst.afw.image.MaskedImageF
+    storage: FitsStorage
+    tables: raw
+    template: focalplane_summary/v%(visit)d-f%(filter)s-focalplane-summary.fits
+  binned_sensor_fits:
+    level: Visit
+    persistable: MaskedImageF
+    python: lsst.afw.image.MaskedImageF
+    storage: FitsStorage
+    tables: raw
+    template: binned_sensor/v%(visit)d-f%(filter)s/R%(raft)s/S%(sensor)s.fits
+  binned_sensor_fits_halves:
+    level: Visit
+    persistable: MaskedImageF
+    python: lsst.afw.image.MaskedImageF
+    storage: FitsStorage
+    tables: raw
+    template: binned_sensor_halves/v%(visit)d-f%(filter)s/R%(raft)s/S%(sensor)s_%(half)s.fits
+  focalplane_summary_png:
```

Comment:
The output files appear to be `GZIP_2` compressed on the level of individual HDUs. This causes trouble for the web fits viewer, and it would be create if we could make sure the HDUs are either uncompressed or `RICE` compressed.